src/main/java/com/gbtec/GenerationRandomizer.java

```diff
@@ -0,0 +1,15 @@
+package com.gbtec;
+
+import java.util.Random;
+
+public class GenerationRandomizer {
+
+    public static void randomizeGeneration(Generation generation, int size) {
+        Random random = new Random();
+        for (int y = 0; y < size; y++) {
+            for (int x = 0; x < size; x++) {
+                generation.getGeneration().get(y).get(x).setStatus(random.nextBoolean());
```

Comment:
If you follow the suggestions above, this line could looks like "generation.getCell(y, x).setStatus(random.nextBoolean());"